docs/usage/administration/monthly_active_users.md

```diff
@@ -0,0 +1,73 @@
+# Monthly Active Users
+
+Synapse can be configured to record the number of monthly active users (also referred to as MAU) on a given homeserver.
+For clarity's sake, MAU only tracks local users.
+
+Please note that the metrics recorded by the [Homeserver Usage Stats](/usage/administration/monitoriing/reporting_homeserver_usage_statistics)
+are calculated differently. The `monthly_active_users` does not take into account any of the rules below, and counts any users 
+who have made a request to the homeserver in the last 30 days.
+
+See the [configuration manual](/usage/configuration/config_documentation.html#limit_usage_by_mau) for details on how to configure MAU.
+
+## Calculating active users
+
+Individual user activity is measured in active days, so if a user performs an action on a given day, that day is then recorded. When
+calculating the MAU figure, any users with a recorded action in the last 30 days are considered part of the cohort.
+
+A user is **never** considered active if they are either:
+ - Part of the trial day cohort (described below)
+ - Have an `appservice_id`.
+
+Otherwise, any request to Synapse will mark the user as active. Internally, this is any request that records
+the client's IP address into the database.
+
+The MAU value is recalculated once every 5 minutes for active users, while inactive users are removed from the cohort once every hour.
+Internally this works by checking all users, and adding any recently active users to the `monthly_active_users` table. Every hour, any
+users with timestamps later than 30 days are removed. The sum of all rows of that table is the final count of active users.
```

Comment:
```suggestion
The MAU value is recalculated once every 5 minutes for active users, while inactive users are removed from the cohort once every hour.
Internally this works by checking all users, and adding any recently active users to the `monthly_active_users` table. Every hour, any
users with timestamps later than 30 days are removed. The count of all rows in that table is the final count of active users.
```

Reply:
I dropped the internally bit, as I think it's not very relevant to administrators **but** can add back if useful.